### PR TITLE
Ros2 v0.8.0 perception launch

### PR DESCRIPTION
**Pull request by TakaHoribe**

### CODE REVIEW

**Comment by s-azumi** [comment]
Did you forget this commit?
https://github.com/tier4/autoware_launcher/commit/4f8a62311a51587193170d6a55090e05ec5f4da2#diff-6670781d62010c894d89d90e0e27810b74903725fbc28b7763e29fb67f03a23b

**Comment by mitsudome-r** [comment]
Fixed in adc7824c572392e247af7164790b95059c1c636b

File: perception_launch/launch/perception.launch.xml [hunk 1]
```diff
@@ -4,17 +4,17 @@
   <!-- common parameters -->
   <arg name="mode" default="camera_lidar_fusion"/>
   <!-- "camera_lidar_fusion", "lidar" or "camera" -->
-  <arg name="image_raw0" default="/sensing/camera/0/image_raw"/>
+  <arg name="image_raw0" default="/sensing/camera/0/image_rect_color"/>
```
Comment (wep21) [inline perception_launch/launch/perception.launch.xml:7]: Single number in namaspace or name is not allowed in ros2.

File: perception_launch/package.xml [hunk 2]
```diff
@@ -18,6 +18,8 @@
   <exec_depend>naive_path_prediction</exec_depend>
   <exec_depend>roi_cluster_fusion</exec_depend>
   <exec_depend>shape_estimation</exec_depend>
+  <exec_depend>tensorrt_yolo</exec_depend>
+  <exec_depend>object_range_splitter</exec_depend>
```
Comment (wep21) [inline perception_launch/package.xml:22]: missing object_merger

Comment (mitsudome-r) [reply perception_launch/package.xml:22]: Fixed in https://github.com/tier4/autoware_launcher.iv.universe/pull/56/commits/cd9edd01c30f959512cf4b1ab816164a3eebb34e

**Review by wep21 — APPROVED** [review]
LGTM

**Pull request merged**